File: SDKs/Aspose.Words-Cloud-SDK-for-Java/src/main/java/com/aspose/words/model/SectionLinkCollectionResponse.java
package com.aspose.words.model;

import com.aspose.words.model.SectionLinkCollection;
public class SectionLinkCollectionResponse {
  private SectionLinkCollection Sections = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getSections
	 * Gets SectionLinkCollection
	 * @return Sections
	 */
  public SectionLinkCollection getSections() {
    return Sections;
  }

	/**
	 * setSections
	 * Sets SectionLinkCollection
	 * @param Sections SectionLinkCollection
	 */
  public void setSections(SectionLinkCollection Sections) {
    this.Sections = Sections;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SectionLinkCollectionResponse {\n");
    sb.append("  Sections: ").append(Sections).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/Aspose.Words.Cloud.Sdk/Model/SectionLinkCollectionResponse.cs
namespace Aspose.Words.Cloud.Sdk.Model 
{
  using System;
  using System.Text;
  using System.Collections;
  using System.Collections.Generic;
  using System.Runtime.Serialization;
  using Newtonsoft.Json;
  using Newtonsoft.Json.Converters;

  /// <summary>
  /// This resonse should be returned by the service when handling:  GET http://api.aspose.com/v1.1/words/Test.doc/sections
  /// </summary>  
  public class SectionLinkCollectionResponse : SaaSposeResponse 
  {                       
        /// <summary>
        /// Collection of sections
        /// </summary>  
        public SectionLinkCollection Sections { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class SectionLinkCollectionResponse {\n");
          sb.Append("  Sections: ").Append(this.Sections).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
